examples/progressbar.py

```diff
@@ -0,0 +1,31 @@
+"""
+ Shows a simple progress bar.
+
+"""
+
+from pylibui.core import App
+from pylibui.controls import Window, ProgressBar
+
+
+class MyWindow(Window):
+
+    def onClose(self, data):
+        super().onClose(data)
+        app.stop()
+
+class MyProgressBar(ProgressBar):
+    pass
```

Comment:
If you are not defining a method with a listener, you don't need to subclass ProgressBar and can use it directly.